libmdnsd/CMakeLists.txt

```
@@ -63,13 +67,33 @@ if(MDNSD_COMPILE_AS_CXX)
 endif()
 
 if(MDNSD_BUILD_OSS_FUZZ)
-    add_library(libmdnsd ${LIBRARY_FILES})
+    add_library(libmdnsd ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
 else()
-    add_library(libmdnsd SHARED ${LIBRARY_FILES})
+    add_library(libmdnsd SHARED ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
 endif()
 
 if(WIN32 OR MINGW)
   target_link_libraries(libmdnsd wsock32 ws2_32)
 endif()
 
 target_compile_definitions(libmdnsd PRIVATE -DMDNSD_DYNAMIC_LINKING_EXPORT)
+
+include(GNUInstallDirs)
+
+install(
+    TARGETS ${PROJECT_NAME}
+    EXPORT ${PROJECT_NAME}Targets
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+)
+
+install(
+    FILES ${PUBLIC_INCLUDES}
+    DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}
+)
+
+install(
+  FILES ${PROJECT_BINARY_DIR}/mdnsd_config.h
+  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}
+)
```

Install the license.
```suggestion
)
install(
    FILES "${PROJECT_SOURCE_DIR}/LICENSE"
    DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/licenses/${PROJECT_NAME}"
) 
```